### PR TITLE
Fail import for packages starting with  minus sign

### PR DESCRIPTION
**Pull request by lukaszachy**
In Beaker metadata this means to not install that package.
However tmt has not place for this in the Test metadata and it should be
done in Plan instead (which cannot be made automatically)

Fixes: #1165

### CODE REVIEW

File: tmt/convert.py [hunk 1]
```diff
@@ -272,19 +272,31 @@ def read_datafile(path, filename, datafile, types, testinfo=None):
             echo(style('environment:', fg='green'))
             echo(pprint.pformat(data['environment']))
 
+    def checked_name(name):
```
Comment (hegerj) [inline tmt/convert.py:275]: just a minor thing, but why did you called this function `checked_name()`?

Comment (lukaszachy) [reply tmt/convert.py:275]: Because the return value is 'name' (of rpm) which is 'checked' ;)
If you have better name....

Comment (hegerj) [reply tmt/convert.py:275]: maybe `negated_` I don't know. Checked doesn't really fit imo. But yeah, it's just a minor thing.

**Review by hegerj — APPROVED** [review]

File: tmt/convert.py (hunk 1 above)
Comment (happz) [reply tmt/convert.py:275]: `sanitized_name`? `normalized_name`? "sanitized" sounds good to me, it's what the function does.

Comment (psss) [reply tmt/convert.py]: Sanitized sounds good, adjusted.

**Review by psss — APPROVED** [review]
Looks good, rebased with some tiny adjustments.

**Pull request merged**